.changeset/blue-frogs-rush.md

```diff
@@ -0,0 +1,26 @@
+---
+'@talend/design-system': major
+---
+
+chore(TUX-1228): update CollapsiblePanel to support multiple actions
+
+## BREAKING CHANGES:
+
+`CollapsiblePanel` now supports multiple actions. The `action` property has been replaced by `actions` which is an array of objects with the same shape used before.
```

Comment:
Just to be sure, do we have some JSON form that use theses fields ?
Cause a breaking change for the TCK form should be avoided

Reply:
Maybe we can avoid breaking changes by don't updating props name and allowing single or multiple values :

`action?: PanelHeaderAction | PanelHeaderAction[];`

And you have to manage then if single or multiple values in CollapsiblePanelHeader.tsx

Reply:
I'm ok with that if @romainseb agrees too

Reply:
I think it's a safer approach indeed

Reply:
Alright, thank you both for your review and suggestions 👍